src/adapter.js

```diff
@@ -11,14 +11,22 @@ const ANNOTATIONS = {
   CANNOT_PARSE: {
     type: 'error',
     code: 1,
+    relation: 'yaml-parser',
   },
   AST_UNAVAILABLE: {
     type: 'warning',
     code: 2,
+    relation: 'yaml-parser',
```

Comment:
I think I may have gotten confused before by this. Here, you call this a relation, but later you use it in the `href` of a link. Wondering if you could name it to something different to make it clearer.